### DISCUSSION TITLE
5.0 release Artefacts

### DISCUSSION THREAD

**Discussion by ashokreddy1208** [Sunbird-Ed/Community#162] [Documentation] [answered]
@luckynvdu 

please share 5.0 release Artefacts to install same on Diksha

cc @chitranshu-keshav  @sharathkashyap @mmanimegalai @anandvarada  @kameshbhr @ShailandraOjha

**Comment by anandvarada** [comment]
@luckynvdu @sharathkashyap as the sign off for release 5.0.0 is completed .. do share the release artefacts on your website so that we can pick it for our instance .. 

cc: @ashokreddy1208  @srajasimman

**Comment by ashokreddy1208** [comment]
@luckynvdu @sharathkashyap  please let us know once 5.0 release artefacts are published on website.

cc @anandvarada @srajasimman

**Reply by luckynvdu** [reply]
The team is working on updating the microsite with release upgrade instructions which should be available by the second half of today. As part of the sign-off, other artifacts have already been shared in the following thread. We are working on making all the artifacts available at the agreed location and they will be in place from the next release onwards

https://github.com/Sunbird-Ed/Community/discussions/161

cc @sharathkashyap @keshavprasadms

**Reply by ashokreddy1208** [reply]
@luckynvdu @sharathkashyap please let us know once 5.0 release artefacts are published on website.

cc @anandvarada @srajasimman

**Comment by sharathkashyap** [comment, marked as answer]
@ashokreddy1208 The release artifacts has been published last Friday. However, we are trying to figure out the issue with gitbook. Meanwhile I have attached the PR which will help you to get tags.
https://github.com/Sunbird-Ed/Community/pull/173/files#diff-aff910fd9e1c53cca10f6ce625f9b5fad4ed411e23c85dc736b6ce69e12b8569

Release Notes are available as part of standard link.
